docs/eventing/getting-started.md

```diff
@@ -0,0 +1,524 @@
+2. Install the Eventing CRDs by running the following command:
+
+    ```sh
+    kubectl apply --selector knative.dev/crd-install=true \
+    --filename https://github.com/knative/eventing/releases/download/{{< version >}}/release.yaml
+    ```
+
+3. Install the Eventing sources by running the following command:
+
+    ```sh
+    kubectl apply --filename https://github.com/knative/eventing/releases/download/{{< version >}}/release.yaml
+    ```
+
+4. Confirm that Knative Eventing is correctly installed by running the following command:
+
+    ```sh
+    kubectl get pods --namespace knative-eventing
+    ```
+    This will return the following result:
+
+    ```sh
+    NAME                                            READY   STATUS    RESTARTS   AGE
+    eventing-controller-6cdf6ff785-44k98            1/1     Running   0          29m
+    eventing-webhook-6c4f6699d8-qclbx               1/1     Running   0          29m
+    imc-controller-85cdb4946b-h2msj                 1/1     Running   0          13m
+    imc-dispatcher-5f4689d868-fspt6                 1/1     Running   0          13m
+    in-memory-channel-controller-d9db9d879-f2jw6    1/1     Running   0          13m
+    in-memory-channel-dispatcher-79bc7f46cd-4mvj6   1/1     Running   0          13m
+    sources-controller-5847564f4f-z59xc             1/1     Running   0          29m
+   ```
+
+## Setting up Knative Eventing Resources 
+
+Before you start to manage events, you need to create the objects needed to transport the events. 
+
+
+### Creating and configuring an Eventing namespace
+
+In this section you create the `event-example` namespace and then add the `knative-eventing-injection` label to that namespace. You use namespaces to group together and organize your Knative resources, including the Eventing subcomponents.
+
+1. Run the following command to create a namespace called `event-example`:
+
+    ```sh
+    kubectl create namespace event-example
+    ```
+
+    This creates an empty namespace called `event-example`.
+
+2. Add a label to your namespace with the following command:
+
+    ```sh
+    kubectl label namespace event-example knative-eventing-injection=enabled
+    ```
+
+   This gives the `event-example`  namespace the `knative-eventing-injection` label, which adds resources that will allow you to manage your events.
+
+In the next section, you will need to verify that the resources you added in this section are running correctly. Then, you can create the rest of the eventing resources you need to manage events.
+
+### Validating that the `Broker` is running
+
+The `Broker` ensures that every event sent by event producers arrives at the correct event consumers. The `Broker` was created when you labeled your namespace as ready for eventing, but it is important to verify that your `Broker` is working correctly. In this guide, you will use the default broker.
```

Comment:
link to https://github.com/knative/docs/blob/master/docs/eventing/broker-trigger.md#Broker